tests/phpunit/integration/tests/Media/Media_Source_Taxonomy.php

```diff
@@ -102,9 +112,9 @@ public function test_rest_api_init() {
 		);
 
 		set_post_thumbnail( $video_attachment_id, $poster_attachment_id );
-		wp_set_object_terms( $video_attachment_id, 'editor', $media_source->get_taxonomy_slug() );
+		wp_set_object_terms( $video_attachment_id, 'editor', $this->instance->get_taxonomy_slug() );
 
-		$request  = new WP_REST_Request( \WP_REST_Server::READABLE, sprintf( '/web-stories/v1/media/%d', $video_attachment_id ) );
+		$request  = new WP_REST_Request( \WP_REST_Server::READABLE, sprintf( '/wp/v2/media/%d', $video_attachment_id ) );
```

Comment:
Why was this changed?

Reply:
As discussed today, this test only needs to test that the field is added to the REST API, not that our custom endpoints are available.